### PR TITLE
Update service.md

### PR DESCRIPTION
**Pull request by HandsomeChoco**
wget url has been changed, it was wrong url 
mattermost-push-proxy-X.X.X.tar.gz => mattermost-push-proxy.tar.gz

### CODE REVIEW

**Comment by neflyte** [comment]
This change applies to releases newer than v5.9 only. It may be useful to document this exception.

Otherwise this looks correct to me; it matches the artifact URLs on the GitHub Releases page.

**Comment by cwarnermm** [comment]
@HandsomeChoco Thank you for your PR. Readers will benefit from knowing that this change applies to mattermost-push-proxy releases v5.9 and later. Are you open to updating your PR to include these details?

**Comment by HandsomeChoco** [comment]
Sure, you can include these details on document.

File: site/content/contribute/mobile/push-notifications/service.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ For the sake of making this guide simple we located the files at `/home/ubuntu/m
 
 1. Download the Mattermost Push Notification Service (any version):
 
-`wget https://github.com/mattermost/mattermost-push-proxy/releases/download/vX.X.X/mattermost-push-proxy-X.X.X.tar.gz`
+`wget https://github.com/mattermost/mattermost-push-proxy/releases/download/vX.X.X/mattermost-push-proxy.tar.gz`
```
Comment (cwarnermm) [inline site/content/contribute/mobile/push-notifications/service.md:27]:
```suggestion
`wget https://github.com/mattermost/mattermost-push-proxy/releases/download/vX.X.X/mattermost-push-proxy.tar.gz (``mattermost-push-proxy`` releases v5.9 and later)`
```

**Review by azigler — APPROVED** [review]
Thanks @HandsomeChoco and @cwarnermm! 🚀

**Pull request merged**